src/plugins/intel_cpu/tests/unit/snippets/precision_propagation_convert_test.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "snippets/pass/propagate_precision.hpp"
+#include "snippets/pass/tokenization.hpp"
+#include "fake_quantize_function.hpp"
+#include "snippets/op/subgraph.hpp"
+#include "ngraph_transformations/snippets_mark_skipped.hpp"
+#include "function_helper.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class PrecisionPropagationConvertTest : public testing::Test {};
+
+TEST_F(PrecisionPropagationConvertTest, smoke_Snippets_PrecisionPropagation_can_be_fused) {
+    const std::set<std::pair<element::Type, element::Type>> precisions_set = {
+        {element::u64, element::u64},
+        {element::u64, element::u32},
+        {element::u64, element::u16},
+        {element::u64, element::u8},
+        {element::u32, element::u32},
+        {element::u32, element::u16},
+        {element::u32, element::u8},
+        {element::u16, element::u16},
+        {element::u16, element::u8},
+        {element::u8, element::u8},
+
+        {element::i64, element::i64},
+        {element::i64, element::i32},
+        {element::i64, element::i16},
+        {element::i64, element::i8},
+        {element::i32, element::i32},
+        {element::i32, element::i16},
+        {element::i32, element::i8},
+        {element::i16, element::i16},
+        {element::i16, element::i8},
+        {element::i8, element::i8},
+
+        {element::f64, element::f64},
+        {element::f64, element::f32},
+        {element::f64, element::f16},
+        {element::f32, element::f32},
+        {element::f32, element::f16},
+        {element::f16, element::f16},
+
+        {element::f32, element::bf16},
+        {element::bf16, element::bf16},
+        {element::f32, element::i8},
+        {element::f16, element::i8},
+        {element::bf16, element::i8},
+        {element::f32, element::u8},
+        {element::f16, element::u8},
+        {element::bf16, element::u8}
+    };
+
+    for (const auto& precisions : precisions_set) {
+        ASSERT_TRUE(ngraph::snippets::pass::PropagatePrecision::can_be_fused(
```

Comment:
Maybe it's better to use `EXPECT_TRUE` here? If I am not mistaken, in a case when we have several failed combinations `ASSERT_TRUE` shows only first fail whereas `EXPECT_TRUE` usually shows all of them.
The same comment for all checks in this file.